modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/ListItems.es.js

```diff
@@ -32,21 +33,40 @@ const ListItems = ({defaultLanguageId, itemType, items}) => {
 	const {
 		dispatch,
 		state: {
-			app: {dataLayoutId, dataListViewId},
+			app: {dataLayoutId, dataListViewId, workflowProcessId},
 		},
 	} = useContext(EditAppContext);
 
-	const itemId = itemType === 'DATA_LAYOUT' ? dataLayoutId : dataListViewId;
+	const itemsProps = {
```

Comment:
@gabrielpramos, What do you think about moving the specific logic for out of this component to make it more generic and putting this properties on the component API?

Reply:
done 👍